tests/python/unittest/test_meta_schedule_tune_relay.py

```diff
@@ -1,554 +0,0 @@
-    weight_np = np.random.uniform(1, 10, size=weight_shape).astype("int8")
-    bias_np = np.random.uniform(1, 10, size=(weight_shape[0],)).astype("int32")
-
-    ref = (
-        relay.create_executor("vm", mod=relay_mod, device=dev, target=target)
-        .evaluate()(*[data, weight_np, bias_np])
-        .numpy()
-    )
-
-    params = {"weight": weight_np, "bias": bias_np}
-
-    if do_tune:
-        extracted_tasks = ms.extract_task_from_relay(relay_mod, target, params)
-        # Filter out tasks that we don't intend to schedule / tune with TIR.
-        tune_tasks = list(
-            filter(
-                lambda task: "dense" in task.task_name,
-                extracted_tasks,
-            )
-        )
-        config = ms.TuneConfig(
-            strategy="replay_trace",
-            num_trials_per_iter=8,
-            max_trials_per_task=8,
-            max_trials_global=8,
-        )
-
-        with tempfile.TemporaryDirectory() as work_dir:
-            # postprocs=lambda: [] is important to prevent default post processors from
-            # tampering with the manual schedule.
-            database = ms.tune_extracted_tasks(
-                tune_tasks,
-                config,
-                work_dir=work_dir,
-                postprocs=lambda: [],
-            )
-    else:
-
-        def schedule_fn(sch) -> bool:
-            if "dense" not in sch.mod.attrs["task_name"]:
-                return False
-
-            block = sch.get_block("compute")
-
-            # Looks up schedule_rule annotation.
-            # See the comment in test_tune_relay_manual_tir_vnni().
-            schedule_rule = sch.get(block).annotations["schedule_rule"]
-
-            assert "dense_vnni" in schedule_rule
-
-            schedule_dense(block, M, False, sch)
-
-            return True
-
-        database = ms.database.ScheduleFnDatabase(schedule_fn)
-
-    with database, tvm.transform.PassContext(
-        opt_level=3,
-        config={"relay.backend.use_meta_schedule": True},
-    ):
-        # pylint: disable=W0105
-        """
-        The log should say
-        Warning: Cannot find workload: tvmgen_default_fused_expand_dims
-        Warning: Cannot find workload: tvmgen_default_fused_cast
-        Warning: Cannot find workload: tvmgen_default_fused_cast_1
-        Warning: Cannot find workload: tvmgen_default_fused_nn_batch_matmul
-
-        This means batch matmul and others are scheduled by TE, and dense (the one not warned)
-        is found in the meta schedule tuning database during compilation
-        """
-        # pylint: enable=W0105
-        lib = relay.build(relay_mod, target=target, params=params)
-
-    runtime = tvm.contrib.graph_executor.GraphModule(lib["default"](dev))
-
-    runtime.set_input("data", data)
-    runtime.run()
-
-    out = runtime.get_output(0).numpy()
-
-    np.testing.assert_equal(out, ref)
-
-
-@tvm.testing.requires_cascadelake
-def test_tune_relay_manual_tir_vnni():
```

Comment:
This test and `test_meta_schedule_relay_lowering` above were not ported to new test files.

Reply:
I migrated them to integration test folder to be more consistent

Reply:
Sorry found the ported test in `unittest/test_meta_schedule_vnni_integration.py`. Didn't notice since the test name changed.